tests/state/test_state_api.py

```diff
@@ -63,3 +64,19 @@ def test_get_states_by_name(self, n, stub_broker, state_middleware, api_client):
         # check integrity
         for state in states:
             assert state in random_states
+
+    def test_request_cancel(self, stub_broker, api_client, state_middleware, cancel_backend):
+        message_id = "3141516"
+        state = State(message_id, StateNamesEnum.Pending, [], {})
```

Comment:
no need for state nor state_middleware, and same for second test. remove it from test arguments too.